### ISSUE TITLE
Review: 2.4.11 Focus Not Obscured (Minimum) (AA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#59]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 2.4.11 considering the mobile research questions (to be completed)

---
When a user interface component receives keyboard focus, the component is not entirely hidden due to author-created content.

Note 1: Where content in a configurable interface can be repositioned by the user, then only the initial positions of user-movable content are considered for testing and conformance of this Success Criterion.

Note 2: Content opened by the user may obscure the component receiving focus. If the user can reveal the focused component without advancing the keyboard focus, the component with focus is not considered hidden due to author-created content.
---
Sufficient Techniques for Success Criterion 2.4.11

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    [C43: Using CSS scroll-padding to un-obscure content ](https://www.w3.org/WAI/WCAG22/Techniques/css/C43.html)

Failures for Success Criterion 2.4.11

    F110: Failure of Success Criterion 2.4.12 Focus Not Obscured (Minimum) due to a sticky footer or header completely hiding focused elements
